src/sqlparser/src/keywords.rs

```diff
@@ -83,6 +83,7 @@ define_keywords!(
     ARRAY_MAX_CARDINALITY,
     AS,
     ASC,
+    ASCII,
```

Comment:
ok, removed

Reply:
Do we need `bit_length` then?

Reply:
> Do we need `bit_length` then?

I think yes, because it appears [here](https://www.postgresql.org/docs/current/sql-keywords-appendix.html). And `OCTET_LENGTH`, the very similar function to `BIT_LENGTH`, which is also in the link, was added in the keyword list before.

Reply:
According to the table, `bit_length` is only a keyword in SQL-92 but not recent standards or PostgreSQL.

Anyways, just making it clear that it is **not a required step** to add keywords when extending our expression/function support. But it is no harm to do here either.

Reply:
OK